docs/operational_guide/multiple_m3db_clusters.md

```diff
@@ -0,0 +1,125 @@
+## Write to multiple M3DB clusters via M3Coordinator
+
+### Overview:
+
+Default M3 architecture has the M3Coordinator writing to and aggregating meterics from a single M3DB cluster. If wanting to add more than one, follow the below instructions. 
+
+Use case(s):
+- Sending metrics to different namespaces for different retention periods, etc.
+
+### Instructions: 
+
+**Note:** Adding mutliple M3DB clusters to m3coordinator using the m3aggregator requires clusterManagement
+
+**Note:** When making API requests, an environment header needs to be set. 
```

Comment:
Maybe say which header here?

Reply:
Gotcha -- i took this note from the original pr pretty sure... what would be an example of a header?

Reply:
@arnikola / @robskillington do you know?